.gitignore

```diff
@@ -182,6 +182,9 @@ todo
 *.asm
 
 # Distribution files
+/dist
```

Comment:
When we release a new version, files in these folders still need to be committed, so please ignore them and just do not commit them instead.